### DISCUSSION TITLE
Receive promise error @click

### DISCUSSION THREAD

**Discussion by yabasha** [alpinejs/alpine#4087] [7. Bugs (archived)]
Hello,

I encounter an error when I click on <div @click="....">. Although it doesn't halt my progress, it is vexing to see this error in the console.

The error originates from `node_modules/alpinejs/dist/module.esm.js` and reads as follows:
"Uncaught (in promise) TypeError: i is not a function at module.esm.js:1089:28."

For reference, I am using the latest build of AlpineJS: v3.13.7.
Browser: MS Edge Version 122.0.2365.80 (Official build) (x86_64)
OS: Mac Sonoma 14.3.1

<img width="427" alt="image" src="https://github.com/alpinejs/alpine/assets/932321/14f52f19-bde2-462b-8665-535769ee2d04">

**Comment by ekwoka** [comment]
What is the expression?

Can you make a simple recreation? ([this might help](https://awesomealpine.com/play))
